datalad/support/gitrepo.py

```diff
@@ -1484,10 +1484,16 @@ def fetch(self, remote=None, refspec=None, progress=None, all_=False,
 
         fi_list = []
         for rm in remotes_to_fetch:
-            fetch_url = \
-                rm.config_reader.get('fetchurl'
-                                     if rm.config_reader.has_option('fetchurl')
-                                     else 'url')
+            from six.moves.configparser import NoOptionError
+            try:
+                fetch_url = \
+                    rm.config_reader.get('fetchurl'
+                                         if rm.config_reader.has_option('fetchurl')
+                                         else 'url')
```

Comment:
Isn't `config_reader` an instance of our own `ConfigManager`? If so, you could just pass a default argument to avoid having to catch an exception.

Reply:
It's not. This still is a `GitConfigParser` from GitPython (property of `git.Remote` "rm").
But should actually work nonetheless and furthermore `ConfigManager` should be used instead, I guess.
So, yes - agree.

Reply:
Actually, this is wrong ;-)
Just noticed, that we both misread that line. It's getting a different variable via that `else`, not a different value of the same variable. So it's not about default value.